plugins/language_erb.lua

```diff
@@ -0,0 +1,55 @@
+-- mod-version:3
+local syntax = require "core.syntax"
+
+syntax.add {
+  name = "html-eruby",
+  files = { "%.htm%.erb$", "%.html%.erb$", "%.erb$" },
```

Comment:
```suggestion
  files = { "%.html?%.erb$", "%.erb$" },
```

Reply:
Make the change. Will make the same in my other PR for 2.0.5 version as well.